examples/syscall.c

```diff
@@ -23,17 +23,49 @@
 #include <linux/sched.h>
 #include <linux/uaccess.h>
 
+
+/* The way we access "sys_call_table" varies as kernel internal changes.
+ * - ver <= 5.4 : manual symbol lookup
+ *   - It works on v5.4 but does not guarantee newer versions.
```

Comment:
The LWN article was mentioned in TeX document, and you don't have to address here.